build/azure-pipelines/darwin/product-build-darwin.yml

```diff
@@ -21,7 +21,7 @@ steps:
 
 - task: NodeTool@0
   inputs:
-    versionSpec: "10.15.1"
+    versionSpec: "12.9.1"
```

Comment:
@deepak1556 maybe coordinate with @joaomoreno, I think it is fair to update node for building when we update Electron, but I think in the past we did so independently from each other.

Reply:
Once this gets merged please do let me know, since there are other places where this comes in:

- https://github.com/microsoft/vscode-linux-build-agent/blob/master/alpine/Dockerfile#L1
- https://github.com/microsoft/vscode-linux-build-agent/blob/master/arm64/Dockerfile#L40
- https://github.com/microsoft/vscode-linux-build-agent/blob/master/armhf/Dockerfile#L40

Reply:
I will revert this change, its better to get this separately. This was only affecting smoke tests for the web step on MacOS.